Source/Windows10/Prism.Autofac.Windows/Prism.Autofac.Windows.csproj

```diff
@@ -14,7 +14,7 @@
     <Version Condition=" '$(PRISM_RELEASE)' != '' ">$(PRISM_RELEASE)</Version>
     <Version Condition=" '$(PRISM_AUTOFAC_WINDOWS_VERSION)' != '' ">$(PRISM_AUTOFAC_WINDOWS_VERSION)</Version>
     <!-- Update this version when incrementing Library Versions i.e. 7.1.0, 7.2.0 -->
-    <VersionPrefix Condition=" '$(PRISM_RELEASE)' == '' And '$(PRISM_AUTOFAC_WINDOWS_VERSION)' == '' ">7.0.0</VersionPrefix>
+    <VersionPrefix Condition=" '$(PRISM_RELEASE)' == '' And '$(PRISM_AUTOFAC_WINDOWS_VERSION)' == '' ">7.1.0</VersionPrefix>
```

Comment:
bumping windows as well?

Reply:
It's just a formality. UWP as we know it is going away.

Reply:
@totht91 we are currently planning on replacing our current Prism for UWP libraries with a merge of Prism and Template10

Reply:
Could you please give us some information about the merge? How can the community help you with this?